handlers/memcached/cluster/ketama.go

```diff
@@ -0,0 +1,120 @@
+package cluster
```

Comment:
can't we reference this from https://github.com/dgryski/go-ketama instead of duplicating?